include/lsst/jointcal/Associations.h

```diff
@@ -34,6 +34,14 @@ class Associations {
   RefStarList photRefStarList; // the (e.g) Landolt stars
   FittedStarList fittedStarList; //  the std::list of stars that are going to be fitted
 
+
+  size_t refStarListSize()
+  { return refStarList.size(); }
+  size_t photRefStarListSize()
+  { return photRefStarList.size(); }
+  size_t fittedStarListSize()
+  { return fittedStarList.size(); }
+
```

Comment:
I assume these are just needed for python wrapping, so I suggest adding them to the .i file. The ideal thing to do would be to template these lists types using std_list.i (if that exists in SWIG) so such lists look like python collections. That's what we do for std::vector. But if std_list.i doesn't work like that or these lists are likely to go away, then it may not be worth the bother.